Shared/R4/resources/Questionnaire-R4-EncounterItems.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "encounter-items",
+  "name": "Encounter Module",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/cqf-questionnaire",
+      "http://hl7.org/fhir/us/davinci-dtr/StructureDefinition/dtr-questionnaire-r4"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicClinicalInfo-prepopulation"
+    }
+  ],
+  "status": "draft",
+  "item": [
+    {
+      "linkId": "ENC.1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".Today"
+          }
+        }
+      ],
+      "text": "Date of encounter (MM/DD/YYYY)",
+      "type": "date",
+      "required": true
+    },
+    {
+      "linkId": "ENC.2",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+          "valueExpression": {
+            "language": "text/cql",
+            "expression": "\"BasicClinicalInfoPrepopulation\".AnswerYes"
+          }
+        },
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "code": "radio-button",
+                "system": "http://hl7.org/fhir/questionnaire-item-control"
+              }
+            ]
+          }
+        }
+      ],
+      "text": "Is this an evaluation of the patient’s need for required device/service/medication?",
```

Comment:
Understand you are trying to generalize the text for this section:

Suggest the following text and let me know if this makes sense:

Is this encounter for the evaluation of the patient’s need for the ordered device, service or medication?

Reply:
Should it be "Is this an encounter ..."?

![Screen Shot 2020-07-17 at 9 19 40 AM](https://user-images.githubusercontent.com/58572522/87790876-453e5900-c80f-11ea-9c60-fc7648a2529b.png)